src/main/java/fr/pilato/elasticsearch/crawler/fs/fileabstractor/FileAbstractorFile.java

```diff
@@ -45,6 +45,11 @@ public FileAbstractModel toFileAbstractModel(String path, File file) {
         model.directory = !model.file;
         model.lastModifiedDate = LocalDateTime.ofInstant(Instant.ofEpochMilli(file.lastModified()), ZoneId.systemDefault());
         model.creationDate = FsCrawlerUtil.getCreationTime(file);
+        try {
```

Comment:
With my comment on FsCrawlerUtil class, we can simply replace all that block with:

```java
model.extension = FsCrawlerUtil.getFileExtension(file);
```